spec/requests/users_controller_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET #index' do
+    before(:example) { get users_path } # get(:index)
+    it 'is a success' do
+      expect(response).to have_http_status(:ok)
+    end
+    it "renders 'index' template" do
+      expect(response).to render_template('index')
+    end
+    it 'includes correct placeholder text' do
+      expect(response.body).to include('List of Users')
```

Comment:
- [ ] Kindly make sure to add tests for GET / SHOW actions for the user controller.